### DISCUSSION TITLE
Can search plugins write (debug) info to the "Execution Log" tab?

### DISCUSSION THREAD

**Discussion by Scare1** [qbittorrent/search-plugins#206] [Q&A]
It would be really handy to be able to do that while developing a search plugin!

**Comment by ackvf** [comment]
Why not run the plugin via command line? https://github.com/qbittorrent/search-plugins/wiki/How-to-write-a-search-plugin#testing-your-plugin

```shell
/qBittorrent/nova3 $ python nova2.py my_plugin all search_keywords
```
